<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>583</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1069</width>
    <height>590</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>XYZ Mail</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/pics/pics/XYZ.ico</normaloff>:/pics/pics/XYZ.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 14px &quot;Microsoft YaHei UI&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>590</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{background-color: rgb(242, 242, 242);}

QPushButton{border:none;}</string>
    </property>
    <widget class="QGraphicsView" name="headpic_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/souce/souce/XYZ.jpg);</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="slidebar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>555</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{background-color: rgb(79, 79, 79);}
QPushButton{
	width:20px;
	height:20px;
}
QPushButton:hover {
       Background-color:rgb(29,171,123);
}</string>
     </property>
     <widget class="QPushButton" name="about">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>关于</string>
      </property>
      <property name="whatsThis">
       <string>关于</string>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">#sliderbar1:hover{
background-color: rgb(255, 170, 127);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/关于 (2).png</normaloff>:/souce/souce/关于 (2).png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="addressbook">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>通讯录</string>
      </property>
      <property name="whatsThis">
       <string>通讯录</string>
      </property>
      <property name="styleSheet">
       <string notr="true">leftbar QPushButtom{background-color:gray;}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/通讯录.png</normaloff>:/souce/souce/通讯录.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="calender">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>日历</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/日历.png</normaloff>:/souce/souce/日历.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="settings">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>设置</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/设置.png</normaloff>:/souce/souce/设置.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QGraphicsView" name="headlogo">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#headlogo{
border-image: url(:/souce/souce/登录.png);
border-radius:40px;
}</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color:rgb(41,189,139);}
QPushButton:hover {
       Background-color:rgb(29,171,123);
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="mainwriteletter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>102</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>写信</string>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/写信.png</normaloff>:/souce/souce/写信.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="mainreceiveletter">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;</string>
      </property>
      <property name="text">
       <string>收信</string>
      </property>
      <property name="icon">
       <iconset resource="souce.qrc">
        <normaloff>:/souce/souce/收信 .png</normaloff>:/souce/souce/收信 .png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="mainlogin">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#mainlogin{
background-color: rgb(41, 189, 139);
border-radius:20px;

}
#mainlogin:hover{
	background-color: rgb(38, 234, 166);
}</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="mainUserName">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>199</y>
       <width>199</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#mainUserName{
font: 87 14px &quot;Arial Black&quot;;
text-decoration: underline;
color:tomato;
}
#mainUserName:hover{
background-color: rgb(85,220,129);
color:white;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="sentBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>410</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#sentBox{background-color: rgb(85, 255, 127);}
#sentBox:hover{
background-color: rgb(123,25,36);
}</string>
     </property>
     <property name="text">
      <string>已发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="receivedBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#receivedBox{background-color: rgb(85, 255, 127);}
#receivedBox:hover{
background-color: rgb(123,25,36);
}</string>
     </property>
     <property name="text">
      <string>收件箱</string>
     </property>
    </widget>
    <widget class="QLabel" name="bufferGif">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>51</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>加载中</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>450</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#deleteBox{background-color: rgb(85, 255, 127);}
#deleteBox:hover{
background-color: rgb(123,25,36);
}</string>
     </property>
     <property name="text">
      <string>已删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="draftBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>490</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#draftBox{background-color: rgb(85, 255, 127);}
#draftBox:hover{
background-color: rgb(123,25,36);
}</string>
     </property>
     <property name="text">
      <string>草稿箱</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="receivedletter">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>260</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#receivedletter{background-color: white;}
#receivedletter QPushButton{
	background-color: rgb(243,243,243);
	border:none;
}
#receivedletter QPushButton:hover{
	color:rgb(29,171,123);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QListWidget" name="emaillist">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>260</width>
       <height>490</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#emaillist{background-color: rgb(242,242,242);}
#emaillist::Item{
width:260px;
height:80px;
}
#emaillist::Item:hover{
background-color: rgb(41, 189, 139);
}</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="sentList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>260</width>
       <height>490</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#sentList{background-color: rgb(242,242,242);}
#sentList::Item{
width:260px;
height:80px;
}
#sentList::Item:hover{
background-color: rgb(41, 189, 139);
}</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="searchList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>260</width>
       <height>490</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#searchList{background-color: rgb(242,242,242);}
#searchList::Item{
width:260px;
height:80px;
}
#searchList::Item:hover{
background-color: rgb(41, 189, 139);
}</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="itemSortMode">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <item>
      <property name="text">
       <string>按时间排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按主题排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按联系人排序</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="itemSortOrder">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>升序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>降序</string>
      </property>
     </item>
    </widget>
    <widget class="QListWidget" name="deleteList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>260</width>
       <height>490</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#deleteList{background-color: rgb(242,242,242);}
#deleteList::Item{
width:260px;
height:80px;
}
#deleteList::Item:hover{
background-color: rgb(41, 189, 139);
}</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="draftList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>260</width>
       <height>490</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#draftList{background-color: rgb(242,242,242);}
#draftList::Item{
width:260px;
height:80px;
}
#draftList::Item:hover{
background-color: rgb(41, 189, 139);
}</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>sentList</zorder>
    <zorder>searchList</zorder>
    <zorder>itemSortMode</zorder>
    <zorder>itemSortOrder</zorder>
    <zorder>deleteList</zorder>
    <zorder>draftList</zorder>
    <zorder>emaillist</zorder>
   </widget>
   <widget class="QGroupBox" name="showemail">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>50</y>
      <width>611</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#showemail{background-color: white;}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWebView" name="emailShow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>82</y>
       <width>611</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="url">
      <url>
       <string>qrc:/souce/index.html</string>
      </url>
     </property>
    </widget>
    <widget class="QGroupBox" name="contInfoTop">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 14px &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="contLogo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/souce/souce/登录.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="contName">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="contEmail">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>邮  箱</string>
      </property>
     </widget>
     <widget class="QLabel" name="contEmailTime">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>时  间</string>
      </property>
     </widget>
     <widget class="QLabel" name="contEmailSubject">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>50</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>主  题</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="attachList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>431</y>
       <width>601</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#attachList{background-color: tomato;
}
#attachList::Item{
height:35px;
width:100px;
}
#attachList::Item:hover{
background-color: rgb(41, 189, 139);
}</string>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>hello world</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>每次看见大客车绝对是.json</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>小撒爱上.jpg</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QLineEdit" name="searchlineEdit">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>19</y>
      <width>190</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#searchlineEdit{background-color:white;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>搜索邮件信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainclose">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>15</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainclose{
	background-color:tomato;
	border-radius:10px;}
#mainclose:hover{border-image: url(:/souce/souce/关闭1.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mainmin">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>15</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainmin{
background-color:rgb(255,224,81);
	border-radius:10px;}
#mainmin:hover{
border-image: url(:/souce/souce/最小化.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mainSearch">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>20</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainSearch{
	border:none;
	border-radius:12px;
}
#mainSearch:hover{
	background-color:rgb(49,126,243);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/搜索框－搜索.png</normaloff>:/souce/souce/搜索框－搜索.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="mainReply">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainReply{
	border:none;
	font-size:14px;
	background-color: rgb(219,219,219);
}
#mainReply:hover{
background-color:rgb(41,189,139);
color:white;
}</string>
    </property>
    <property name="text">
     <string>回复</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/发送.png</normaloff>:/souce/souce/发送.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="mainForward">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainForward{
	border:none;
	font-size:14px;
	background-color: rgb(219,219,219);
}
#mainForward:hover{
background-color:rgb(41,189,139);
color:white;
}</string>
    </property>
    <property name="text">
     <string>转发</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/发送.png</normaloff>:/souce/souce/发送.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="delEmail">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#delEmail{
	border:none;
	font-size:14px;
	background-color: rgb(219,219,219);
}
#delEmail:hover{
background-color:tomato;
color:white;
}</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/关闭1.png</normaloff>:/souce/souce/关闭1.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="mainAttach">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainAttach{
	border:none;
	font-size:14px;
	background-color: rgb(219,219,219);
}
#mainAttach:hover{
background-color:tomato;
color:white;
}</string>
    </property>
    <property name="text">
     <string>查看附件</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/附件.png</normaloff>:/souce/souce/附件.png</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="searchMode">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>请选择</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>主题</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>时间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>联系人</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>邮件内容</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="restoreEmail">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#restoreEmail{
	border:none;
	font-size:14px;
	background-color: rgb(219,219,219);
}
#restoreEmail:hover{
background-color:tomato;
color:white;
}</string>
    </property>
    <property name="text">
     <string>还原</string>
    </property>
    <property name="icon">
     <iconset resource="souce.qrc">
      <normaloff>:/souce/souce/还原.png</normaloff>:/souce/souce/还原.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="souce.qrc"/>
 </resources>
 <connections/>
</ui>
